src/App.js

```diff
@@ -11,6 +13,8 @@ function App() {
         <Routes>
           <Route path="/" element={<Homepage />} />
           <Route path="/timerpage" element={<TimerPage />} />
+          <Route path="/settings" element={<Settings />} />
```

Comment:
`/` can be omitted in route path for react router v6

Reply:
Done